commit 7081c91bcf61086a4a5ce6a269425ea5f682a95f (refs/changes/24/3924/2)
Author: Jason King <jason.king@joyent.com>
Date:   2018-05-09T15:44:35-05:00 (1 year, 5 months ago)
    
    OS-6943 varpd not listed as an install_h target
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/lib/Makefile b/usr/src/lib/Makefile
index 7d337e7175..9efd87bc10 100644
--- a/usr/src/lib/Makefile
+++ b/usr/src/lib/Makefile
@@ -506,6 +506,7 @@ HDRSUBDIRS=				\
 	smbsrv		\
 	smhba		\
 	udapl		\
+	varpd		\
 	$($(MACH)_HDRSUBDIRS)
 
 i386_HDRSUBDIRS=	\
